packages/testing-helpers/README.md

```diff
@@ -83,3 +83,21 @@ await nextFrame();
 // await aTimeout(10); // would wait 10ms
 expect(el.shadowRoot.querySelector('#foo').innerText).to.equal('baz');
 ```
+
+## Fixture Cleanup
+By default, if you import anything via `import { ... } from '@open-wc/testing-helpers';`, it will automatically register a side-effect that cleans up fixtures.
```

Comment:
```suggestion
By default, if you import anything via `import { ... } from '@open-wc/testing-helpers';`, it will automatically register a side-effect that cleans up your fixtures.
```